src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -452,23 +461,36 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        String instructorFragmentArgumentInstructors = generateInstructorFragment(course.getId(), course.getName());
+
+        String instructorFragmentArgumentStudents = "";
+
         for (InstructorAttributes instructor : instructors) {
-            emails.add(generateFeedbackSessionClosedEmail(course, session, instructor.name, instructor.email));
+            emails.add(generateFeedbackSessionClosedEmail(course, session, instructor.name, instructor.email,
+                    instructorFragmentArgumentInstructors));
         }
         for (StudentAttributes student : students) {
-            emails.add(generateFeedbackSessionClosedEmail(course, session, student.name, student.email));
+            emails.add(generateFeedbackSessionClosedEmail(course, session, student.name, student.email,
+                    instructorFragmentArgumentStudents));
         }
 
         return emails;
     }
 
+    /**
+     * Generates the feedback session closed email for both
+     * the instructors and the students with the given
+     * {@code userName} and {@code userEmail}.
```

Comment:
Looks like these three lines can be packed into two lines instead. A line can go as far as 120 chars.